packages/api/methods.test.ts

```diff
@@ -346,6 +346,166 @@ describe('API CRUD operations', () => {
     );
   });
 
+  // apis: getPayeeRules, createPayeeRule, updatePayeeRule, deletePayeeRule
+  test('PayeeRules: successfully update payee rules', async () => {
+    await api.createPayee({ name: 'test-payee' });
+    await api.createPayee({ name: 'test-payee2' });
+
+    // create our test rules
+    const rule = {
+      id: null,
+      stage: 'pre',
+      conditionsOp: 'and',
+      conditions: [
+        {
+          field: 'payee',
+          op: 'is',
+          value: 'test-payee',
+        },
+      ],
+      actions: [
+        {
+          op: 'set',
+          field: 'category',
+          value: 'fc3825fd-b982-4b72-b768-5b30844cf832',
+        },
+      ],
+    };
+    const ruleId = await api.createRule(rule);
+    rule.id = ruleId;
```

Comment:
This feels super weird.  I had to add an `id: null` into the object when creating it and then patch the ID back in.  But as far as I can tell, the other creates just return a string ID so this is mimicking that behavior.